projects/fsastorefront-e2e-cypress/cypress/helpers/fnol.ts

```diff
@@ -233,12 +221,119 @@ export function checkDownloadButton() {
   });
 }
 
-export function checkClaimReplication() {
+export function checkClaimReplication(incidetType) {
   cy.get('.info-card')
     .should('have.length', 1)
     .within(() => {
       cy.get('.info-card-data').within(() => {
-        cy.get('.value').should('contain.text', 'Collision');
+        cy.get('.value').should('contain.text', incidetType);
       });
+      cy.get('div.info-card-links').contains('Details');
```

Comment:
No need for `div` in selector. Please stop using this, and refactor all occurrences